p2p/src/network.rs

```diff
@@ -59,6 +59,8 @@ where
     pub new_peers: HashMap<ConnectionId, Addr<Peer<T, K, E>>>,
     /// Current [`Peer`]s in [`Peer::Ready`] state.
     pub peers: HashMap<PublicKey, RefPeer<T, K, E>>,
+    /// Connecting to peer id's.
+    connecting_peer_keys: Vec<PublicKey>,
```

Comment:
We are only removing values from this field and never add anything, what purpose this field has?